solana/caip2.md

```diff
@@ -54,13 +56,13 @@ To resolve a blockchain reference for the Solana namespace, make a JSON-RPC requ
 {
   "id": 1,
   "jsonrpc": "2.0",
-  "result": "4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZAMdL4VZHirAn"
+  "result": "5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d"
 }
 ```
 
-The response will return as a value for the result a hash for the block with
-height 0 that should be truncated to its first 32 characters to be CAIP-30
-compatible.
+The response will return as a value for the result a 44-character
+Base-58-encoded hash for the block with height 0 that should be truncated to its
```

Comment:
Do we care to distinguish base58btc from base58flickr?

https://github.com/multiformats/multibase/blob/master/multibase.csv

| encoding | code | description | status |
| -- | -- | -- | -- |
| base58btc | z | base58 bitcoin | default |
| base58flickr | Z | base58 flicker | candidate |

| base | alphabet |
| -- | -- |
| base58btc | `"123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"` |
| base58flickr | `"123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ"` |

I believe it base58btc but didn't look it up.

Reply:
That is a very good question!
I am deducing that they use base58btc from the fact that [the crate they depend on](https://crates.io/crates/solana-program/1.8.6/dependencies) for base58 en/decoding is `[bs58](https://crates.io/crates/bs58/0.3.1)`, which allows multiple encoding alphabets but defaults to "the BITCOIN alphabet" ([src](https://docs.rs/bs58/0.3.1/bs58/alphabet/constant.DEFAULT.html)).  Will update the text and references now, since this could be determinant for implementers.

Reply:
this is also interesting, on the subject of base58btc and checksums:
https://github.com/solana-labs/solana/issues/6970

Reply:
resolving convo cuz PR updated